components/home/integrationsHead.tsx

```diff
@@ -12,10 +12,10 @@ export default function IntegrationsHead({
   return (
     <>
       <Head>
-        <title>{name} Integrations - Best Data App Integration Tools</title>
+        <title>{name} Data Integration ETL Tools - Import &#38; Upload</title>
```

Comment:
I actually have a question out with the SEO agency about this one. They made general recommendations to ~all integrations pages which I already merged, but this Salesforce one was an outlier. Mind if we revert this change for now until they get back to me? Sorry about that!